app/src/main/java/com/fluffybros/tujefe/ui/add/ManualAddFragment.kt

```diff
@@ -20,9 +20,8 @@ class ManualAddFragment : Fragment(R.layout.fragment_manual_add) {
 
         binding.manualAddButton.setOnClickListener {
             mainViewModel.addRecyclerItem(binding.nameField.editText?.text.toString(), binding.setupKeyField.editText?.text.toString())
-            val imm: InputMethodManager =
-                requireContext().getSystemService(Activity.INPUT_METHOD_SERVICE) as InputMethodManager
-            imm.hideSoftInputFromWindow(view.windowToken, 0)
+            val imm = ContextCompat.getSystemService(view.context, InputMethodManager::class.java)
```

Comment:
you don't need ContextCompat, since our `minSdkVersion` is so high -

you can just do `val imm = requireContext().getSystemService<InputMethodManager>()`